GoogleUtilities/Example/Tests/Swizzler/GULAppDelegateSwizzlerTest.m

```diff
@@ -431,7 +632,7 @@ - (void)skipped_testAppDelegateInstance {
   GULTestAppDelegate *realAppDelegate = [[GULTestAppDelegate alloc] init];
```

Comment:
Is `skipped_testAppDelegateInstance` still disabled due to flakiness? If it is, can you make sure it passes locally?

Reply:
@tejasd  TBH, I didn't really pay attention to the skipped test, just kept it compilable... If you have a context why it has been marked as skipped? It looks like you were the last person changing it

Reply:
It was because it was flaky on Travis! (Works ok on the internal infrastructure, so should be caught when you copybara it)

https://github.com/firebase/firebase-ios-sdk/pull/1564/commits/cb73b2f1bdb51bfea5b8b790188423e0b99a7170